src/crc.rs

```diff
@@ -69,8 +69,11 @@ impl Hasher {
 
     pub fn reset(&mut self) {
         self.val.reset();
-        for x in self.bytes.iter_mut() {
-            *x = 0;
+        #[cfg(not(feature = "nightly"))]
+        {
+            for x in self.bytes.iter_mut() {
+                *x = 0;
+            }
```

Comment:
`self.bytes` doesn't need to be reset to 0. It's just a buffer that we need in order to be able to reverse all bits given to the hasher before sending them to the CRC32B `crc32fast::Hasher`

Reply:
Yeah, not sure why I added that in. Deleting.